scripts/bulk_index.pl

```diff
@@ -43,12 +43,10 @@ BEGIN
 }
 
 my $indexer = Bugzilla::Elastic::Indexer->new(
-    $debug_sql ? ( debug_sql => 1 ) : (),
+    $debug_sql    ? ( debug_sql => 1 )                      : (),
     $progress_bar ? ( progress_bar => 'Term::ProgressBar' ) : (),
 );
 
-$indexer->create_index;
```

Comment:
It seems like this removal pairs with an above code change that auto-creates indexes now?